openedx/features/edly/utils.py

```diff
@@ -373,6 +373,19 @@ def get_current_site_invalid_certificate_context(default_html_certificate_config
     context['company_tos_url'] = get_tos_and_honor_code_url()
     return context
 
+
+def get_logo_from_current_site_configurations():
+    """
+    Return the `logo_src` value from current site configurations.
```

Comment:
The function is returning context with logo_src in it. Shouldn't the comment be updated?

Reply:
improved the docstring.